src/app/components/plugin-sidebar/plugin-sidebar.component.ts

```diff
@@ -101,6 +101,46 @@ export class PluginSidebarComponent implements OnInit, OnDestroy {
                 this.switchActiveTemplateLink(template?.self ?? null);
             }
         });
+        this.registry.newApiObjectSubject
```

Comment:
Add a newline (and maybe a comment) above each new observable pipe/subscription start to make this method more readable. This is not required for smaller methods with one or two such observable chains, but really helps when there are more of them in a single method.